OpenTelemetry.sln

```diff
@@ -11,6 +11,7 @@ Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "Solution Items", "Solution
 	ProjectSection(SolutionItems) = preProject
 		.dockerignore = .dockerignore
 		.editorconfig = .editorconfig
+		build\concurrency-tests-with-Coyote.ps1 = build\concurrency-tests-with-Coyote.ps1
```

Comment:
Consider https://www.interfacett.com/blogs/how-should-i-name-my-powershell-scripts/.

Reply:
does "test-threadSafety" make more sense?

Reply:
> does "test-threadSafety" make more sense?

👍